packages/typespec-test/test/NetworkAnalytics.Management/tspconfig.yaml

```
@@ -1,6 +1,8 @@
 emit: ["@azure-tools/typespec-ts"]
 options:
   "@azure-tools/typespec-ts":
+    typespecTitleMap: 
+        NetworkAnalyticsClient: NetworkAnalyticsManagementClient
```

```suggestion
        NetworkAnalyticsClient: NetworkAnalyticsManagementApi
```
Change it into a non Client suffix so that it can better reflect when typespecTitleMap is a better option.